Microsoft.Toolkit.Win32.UI.Controls/Interop/WinRT/WebViewControlProcessCapabilityState.cs

```diff
@@ -0,0 +1,28 @@
+// ******************************************************************
+// Copyright (c) Microsoft. All rights reserved.
+// This code is licensed under the MIT License (MIT).
+// THE CODE IS PROVIDED “AS IS”, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+// INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+// IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+// TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH
+// THE CODE OR THE USE OR OTHER DEALINGS IN THE CODE.
+// ******************************************************************
+
+namespace Microsoft.Toolkit.Win32.UI.Controls.Interop.WinRT
+{
+    // Type is a copy. Information regarding the origination of the type is in summary comments
+#pragma warning disable CS1591 // Missing XML comment for publicly visible type or member
```

Comment:
Why?

Reply:
Documentation should be taken from `Windows.Web.UI.Interop.WebViewControlProcessCapabilityState`. If we duplicate here, we must maintain two copies of that documentation. Items in the WinRT namespace that are public have the same types of remarks and `seealso` so as to direct the reader to the authoritative documentation source.